src/visualization/types/Graph/view.tsx

```diff
@@ -253,4 +272,29 @@ const XYPlot: FC<Props> = ({
   return <Plot config={config} />
 }
 
-export default XYPlot
+const mapStateToProps = (state: AppState, ownProps: OwnProps) => {
+  // the view inside a cell for which we want to update the view properties of
+  const view = getByID<View<XYViewProperties>>(
+    state,
+    ResourceType.Views,
+    ownProps.cellID
+  )
+
+  const {
+    timeMachines: {activeTimeMachineID},
+  } = state
+
+  const veoOpen = activeTimeMachineID === 'veo'
+
+  return {
+    view,
+    veoOpen,
+  }
+}
+
+const mapDispatchToProps = {
+  saveViewPropertiesToIDPE: updateViewAndVariables,
+}
+
+const connector = connect(mapStateToProps, mapDispatchToProps)
+export default connector(XYPlot)
```

Comment:
since this is a function component, search for how `useReducer` is used, including the use of `dispatch` with `useReducer`. Using `useReducer` will take the place of both `mapStateToProps` and `mapDispatchToProps` in this component.